src/YesSql.Core/Commands/BachCommand.cs

```diff
@@ -62,7 +62,13 @@ public async Task ExecuteAsync(DbConnection connection, DbTransaction transactio
                     logger.LogWarning("The default capacity of the BatchCommand StringBuilder {Default} might not be sufficient. It can be increased with BachCommand.DefaultBuilderCapacity to at least {Suggested}", DefaultBuilderCapacity, command.Length);
                 }
 
-                var count = await connection.ExecuteAsync(command, Parameters, transaction);
+                using (var dr = await connection.ExecuteReaderAsync(command, Parameters, transaction))
+                {
+                    foreach (var action in Actions)
+                    {
+                        action(dr);
+                    }
+                }
```

Comment:
You could use `using var dr` instead of the `using` block.

Reply:
I prefer to use blocks to make the boundaries visible, just a personal choice here

Reply:
Ah OK.